webapp/src/components/AddressErrorPage.js

```diff
@@ -18,23 +16,23 @@ export default function AddressErrorPage() {
         <p>
           Remember that you to participate in the auction
           <br />you need to have MANA staked on the&nbsp;
-          <Link to="https://terraform.decentraland.org" target="_blank">
+          <a href="https://terraform.decentraland.org" target="_blank" rel="noopener noreferrer">
```

Comment:
I don't get the reason for this change. If the problem is not having `rel="noopener noreferrer"` on the `<a>` they can be added.

This:

```jsx
<Link to="https://terraform.decentraland.org" target="_blank" rel="noopener noreferrer">
  Terraform Registration
</Link>
```

renders this

```html
<a target="_blank" rel="noopener noreferrer" href="https://terraform.decentraland.org">Terraform Registration</a>
```

Reply:
Same question applies for the rest of the error pages

Reply:
https://github.com/ReactTraining/react-router/issues/1147